### DISCUSSION TITLE
Google日历网站乱码

### DISCUSSION THREAD

**Discussion by hnboy2005** [F9y4ng/GreasyFork-Scripts#125] [问答专区(Question / Answer)]
<img width="1549" alt="image" src="https://user-images.githubusercontent.com/3166227/188054692-0701e69b-9387-472e-add6-7c9b4fbb0207.png">

**Comment by hnboy2005** [comment]
正常应该如下
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/3166227/188054842-c4d8276a-81eb-4d7e-99b5-81a677c14156.png">

**Comment by F9y4ng** [comment]
在 **需要渲染的网页元素** 中，追加 `:not(span[class][aria-hidden='true'])`，保存为站点独享。

## 额外的建议
* 使用最新版本的脚本。
* 我看你截图中的配置是旧版本的默认参数，建议你先 **重置** 所有默认参数，以应用最新的过滤规则，然后再重新设置保存你的设置。
* 其他独享参数及全局参数如有遇到样式异常也建议先按以上操作执行。
